django-data/image/submissions/tests/test_view_validation_summary.py

```diff
@@ -0,0 +1,49 @@
+from django.test import Client, TestCase
+from django.urls import resolve, reverse
+
+from common.tests import GeneralMixinTestCase, OwnerMixinTestCase
+
+from ..views import SubmissionValidationSummaryView
+
+
+class SubmissionValidationSummaryViewTest(TestCase):
+    """Test Submission DetailView"""
+    fixtures = [
+        "image_app/user",
+        "image_app/dictcountry",
+        "image_app/dictrole",
+        "image_app/organization",
+        "image_app/submission"
```

Comment:
You could also create a fixture for SubmissionSummary and try to determine if your data are submitted to the views

Reply:
As I've added fixture there is no need to have create_vcsummary patch in tests.

Reply:
Done